src/index.js

```diff
@@ -273,13 +275,16 @@ module.exports = {
         inputs,
       });
 
+      console.log('------', audits);
```

Comment:
@aitchiss I noticed this appearing in the deploy logs, not sure if it was intentional?! 🙃

Reply:
🙈